.github/workflows/build-push-and-deploy-container.yml

```diff
@@ -5,6 +5,7 @@ on:
   push:
     branches:
       - main
+      - docker-build-push-action-v2
```

Comment:
I just added this line so that we could test it running in situ - please feel free to remove it when it's ready to merge